packages/composables/src/getters/checkoutGetters.ts

```diff
@@ -1,16 +1,14 @@
 import { ShippingMethod } from '@vue-storefront/spree-api';
+import { ShippingRate } from '@vue-storefront/spree-api/src/types';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getShippingMethodId = (shippingMethod: ShippingMethod): string => '';
+export const getShippingMethodId = (shippingMethod: ShippingRate): string => shippingMethod.id;
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getShippingMethodName = (shippingMethod: ShippingMethod): string => '';
+export const getShippingMethodName = (shippingMethod: ShippingRate): string => shippingMethod.name;
```

Comment:
Looking at e.g. productGetters - shouldn't we also handle shippingMethod=undefined here? And in such case return empty string or something?
The same for other getters :)

Reply:
I think `shippingMethod` shouldn't be undefined (if there are no shipping methods - we are not calling these getters), although we can add that to be 100% sure